requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytest~=7.4.0
```

Comment:
This file is unnecessary, dependencies are already specified under dev_tools/requirements.